.github/workflows/checks.yml

```
@@ -13,26 +13,15 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Set up Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: "20"
+          node-version: "22"
 
-      - name: Install dependencies
-        run: npm ci
+      - name: get prettier prettier-plugin-svelte
+        run: npm ci --save-dev prettier-plugin-svelte prettier
```

```suggestion
        run: npm ci
```
Det er best å ha dette declarative i package.json uansett om vi bruker install eller ci, heller enn å imperatively gjøre det her.

---

Hva tenker du? Man trenger ikke hele package json her.

---

> Hva tenker du? Man trenger ikke hele package json her.

Både npm install og npm ci installerer hele package.json uansett. Å skrive dette (på begge måter) vil installere først alt i package.json, og så de tingene du spesifiserte. Det er ingen negative konsekvenser for oss å installere hele package.json, mens det blir mer jobb å kun installere spesifikke ting, i tillegg til mulige versjonskonflikter.

---

Du har veldig rett. Jeg testa på en local runner og tok tiden, men fekk ikke med meg output